test/Date/DateParse.js

```diff
@@ -174,3 +174,58 @@ WScript.Echo(Date.parse("2011/11-08 19:48:43"));
 
 // Multiple 'T's aren't allowed
 WScript.Echo(Date.parse("2011-11-08TT19:48:43"));
+
+// Verifying non-standard delimiters with milliseconds
+
+// Missing digits after . or :
+WScript.Echo(Date.parse("2011-11-08 19:48:43."));

```

Comment:
I personally really dislike these types of tests, because I have no idea if the baseline contains the right answer or not.
I would prefer something like
```js
function doTest(format, expected) {
  if (Date.parse(format) !== expected) {
    console.log(`Invalid date parsing result, expected ${format} to give ${expected}`);
  }
}

// Notice this test shows the expected result which doesn't match the current baseline output
doTest("2011-11-08 19:48:43:1", 1320810523001);
...
```

Reply:
I agree, but this whole test has that problem.

Also having a multiplied-out value showing the represented value would add more semantic meaning to the expected output.

The tests being added can use the better style but fixing the rest of the test is out of scope and up to @xiaoyinl whether to update.